Userland/Libraries/LibJS/Runtime/RegExpObject.h

```diff
@@ -49,13 +49,16 @@ class RegExpObject : public Object {
     String const& flags() const { return m_flags; }
     Regex<ECMA262> const& regex() { return *m_regex; }
     Regex<ECMA262> const& regex() const { return *m_regex; }
+    bool has_legacy_features() const { return m_has_legacy_features; }
+    void set_has_legacy_features(bool flag) { m_has_legacy_features = flag; }
```

Comment:
The slot is called `LegacyFeaturesEnabled`, so:

```suggestion
    bool legacy_features_enabled() const { return m_legacy_features_enabled; }
    void set_legacy_features_enabled(bool legacy_features_enabled) { m_legacy_features_enabled = legacy_features_enabled; }
```

(`flag` is not a good name here because it has a different meaning for regex)